app/monitoring/Tip.scala

```diff
@@ -54,6 +62,19 @@ object PathVerification {
     case _ => None
   }
 
+  def monitoredRegion(country: Country): Option[MonitoredRegion] = country match {
+    case Country.Australia => Some(AU)
```

Comment:
What's the difference between `Country` and `MonitoredRegion`?

Reply:
`Country` is used to model any country which can be selected as a billing or shipping address (makes more sense in the context of subscriptions), whereas `MonitoredRegion` only applies to higher traffic countries/flows we want to include on this dashboard.